requirements.txt

```diff
@@ -2,5 +2,7 @@ numpy
 scipy
 scikit-image
 boto
+pillow
```

Comment:
hm, no, i guess i meant, if it's included in `scikit-image`, we shouldn't need the explicit dependency right?